### ISSUE TITLE
Add a links to julialang.org

### ISSUE THREAD

**Issue by tordans** [JuliaGeo/meta#18]
I could not find out what Julia is anywhere on https://github.com/JuliaGeo, https://juliageo.org/ or the Repos I looked at. 

I suggest linking to https://julialang.org/  the first time "Julia" (as in "the Language") is used. Eg on the Github Page and in the various READMEs.

**Comment by Alexander-Barth** [comment]
I agree, I have also added these to links to the package that I maintain (CFTime and GeoDatasets).

**Comment by fbanning** [comment]
This is a good idea for newcomers to Julia and JuliaGeo. I've taken the liberty to make a PR adding such a link.

**Comment by visr** [comment]
Thanks @fbanning! The issue also suggests to do the same for the README.md's of the JuliaGeo repos, which I agree is a good suggestion. Are you feeling like going over some repos as well?
